fossunited/ticketing/doctype/foss_event_ticket_transfer/test_foss_event_ticket_transfer.py

```diff
@@ -24,12 +26,31 @@ def test_ticket_transfer(self):
         )
         event.insert()
 
+        self.event = event
+
+    def tearDown(self):
+        frappe.delete_doc("FOSS Chapter Event", self.event.name, force=True)
```

Comment:
One question, 

Why is this method deleting the document which is going to be a event ?

Reply:
A good read: https://hands-on.cloud/python-unit-tests/

> Python Unit Test Teardown
 The counterpart to the setup process in unit testing is the ‘teardown’ process. The tearDown() method in the unittest module is where you clean up any resources set up. This method is automatically called after each test.